cluster/gce/util.sh

```diff
@@ -807,7 +804,16 @@ function create-loadbalancer() {
     --target-pool ${MASTER_NAME} --address=${KUBE_MASTER_IP} --ports=443
 
   echo -n "Waiting for the load balancer configuration to propagate..."
-  until $(curl -k -m1 https://${KUBE_MASTER_IP} > /dev/null 2>&1); do echo -n .; done
+  local counter=0
+  until $(curl -k -m1 https://${KUBE_MASTER_IP} &> /dev/null); do
+    counter=$((counter+1))
+    echo -n .
+    if [[ ${counter} -ge 1800 ]]; then
+      echo -e "${color_red}TIMEOUT${color_norm}" >&2
+      echo -e "${color_red}Load balancer failed to initialize within ${counter} seconds.${color_norm}" >&2
+      exit 2
+    fi
```

Comment:
During this loop those requests will timeout after 1s, so they will not run more often than once every second. Do you think we should make it less often?